mods/cnc/rules/structures.yaml

```diff
@@ -851,10 +851,15 @@ GUN:
 		HasMinibib: true
 	Turreted:
 		TurnSpeed: 48
-		InitialFacing: 224
+		InitialFacing: 192
 		RealignDelay: -1
-	-WithSpriteBody:
-	WithEmbeddedTurretSpriteBody:
+		RequiresCondition: !build-incomplete
+	WithSpriteBody:
```

Comment:
`WithSpriteBody` is already inherited from `^Defense` so no need to define it again here or in RA.